doc/ignore-rule.md

```diff
@@ -0,0 +1,62 @@
+## Ignoring a rule
+
+
+> [!NOTE]
+> TL;DR: use the command `dd-no-sa` to ignore a rule
+
+
+### Ignoring all results in a file
+
+To ignore all rules in a file, put `no-dd-sa` in a comment
+on the **first line** of the file.
+
+In the following code snippet, all rules are ignored in the file.
+
+```python
+#no-dd-sa
```

Comment:
It will then confuse the syntax highlighting. And there is no way to specify line numbers in GitHub. See [this](https://stackoverflow.com/questions/36122803/how-to-display-line-numbers-using-github-flavored-markdown-code).